### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Sites-Create Web Site_Screen Reader: Narrator/NVDA is reading group after "Create Web Site" button, even if there are no controls present on the page.

### ISSUE THREAD

**Issue by accessibility18** [Microsoft/IIS.WebManager#138]
**Environment Details:**
URL: https://manage.iis.net/
Browser: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106
Screen Reader: Narrator, NVDA

**MAS ID Violated:** MAS1.3.2

**Repro Steps:** 
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till "Web Sites" in the left menu section and Press Enter.
5. Tab till "Create Web Site" expandable button and Press Enter.
6. Verify that screen reader is reading all the controls of the page in a proper sequence.

**Actual Result:**
Narrator is reading group and NVDA is reading as "clickable clickable" after "Create Web Site" button, even if there are no controls present on the page.

**Expected Result:**
Screen reader focus should move to the controls or contents that are present on the page. It should not read any hidden or programmatic items that are not available at the UI for the users.

**User Impact:**
Screen Reader users may think it as a different control and they may try to access it. But since there are no UI components available at that point, user will not get any result of the user action. So this will create an inconvenience to the visually impaired users who are using screen reader.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={1ee227a7-e3c5-47b8-ab18-48ca8595077b}

**Attachment:**
[Bug 138_Screen reader is reading unavailable control at UI as group.pptx](https://github.com/Microsoft/IIS.WebManager/files/2327886/Bug.138_Screen.reader.is.reading.unavailable.control.at.UI.as.group.pptx)

**Comment by jhkimnew** [comment]
Fixed

**Comment by mstechie** [comment]
GitHubTags:#A11y_Manage.IIS.Net;
